### PR TITLE
RFE: Support dummy interfaces

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

**Review by cathay4t — CHANGES_REQUESTED** [review]
Integration test case required.

**Comment by liangwen12year** [comment]
> Integration test case required.
Got it!

File: tests/playbooks/tests_dummy.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+              - tasks/assert_device_absent.yml
+            lsr_test:
+              - tasks/create_dummy_profile.yml
+            lsr_assert:
+              - tasks/assert_profile_present.yml
+              - tasks/assert_device_present.yml
+            lsr_cleanup:
+              - tasks/cleanup_profile+device.yml
+      tags:
+        - "tests::cleanup"
```
Comment (tyll) [inline tests/playbooks/tests_dummy.yml:30]: cleanup seems to be wrong here

Comment (liangwen12year) [reply tests/playbooks/tests_dummy.yml:30]: got it. I found the root cause of the failure. 

`nmcli con add type dummy ifname dummy0 ipv4.addresses 192.0.2.42/30` will never create a profile in `/etc/sysconfig/network-scripts/`

Maybe we can change the tests file to exclude `- tasks/assert_profile_present.yml`

**Review by cathay4t — APPROVED** [review]

**Pull request merged**